bundle/src/main/java/com/adobe/acs/commons/workflow/synthetic/cq/WrappedSyntheticWorkItem.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2016 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package com.adobe.acs.commons.workflow.synthetic.cq;
+
+public interface WrappedSyntheticWorkItem {
```

Comment:
Do these require a package-info version bump? These are exported right?

Reply:
I suppose that this only a minor change at most because these interfaces aren't used by any existing exported type signatures. @froggin we should probably at least add @ProviderType to these Wrapped interfaces.

Reply:
this is the maven-bundle-plugin:3.4.0:baseline output on a recent version of the master branch, which might indicate that child packages of workflow.synthetic aren't exported: 
```
[INFO] -----------------------------------------------------------------------------------------------------------
[INFO]   com.adobe.acs.commons.workflow                     unchanged  1.1.1      1.1.1      1.1.1      -
[INFO] -----------------------------------------------------------------------------------------------------------
[INFO]   com.adobe.acs.commons.workflow.bulk.execution      unchanged  1.0.1      1.0.1      1.0.1      -
[INFO] -----------------------------------------------------------------------------------------------------------
[INFO]   com.adobe.acs.commons.workflow.bulk.execution.model unchanged  4.0.0      4.0.0      4.0.0      -
[INFO] -----------------------------------------------------------------------------------------------------------
[INFO]   com.adobe.acs.commons.workflow.bulk.removal        unchanged  4.0.0      4.0.0      4.0.0      -
[INFO] -----------------------------------------------------------------------------------------------------------
[INFO]   com.adobe.acs.commons.workflow.synthetic           unchanged  3.0.0      3.0.0      3.0.0      -
[INFO] -----------------------------------------------------------------------------------------------------------
```

Reply:
I just added the `@ProviderType` annotations, and put the ResourceResolver back in the adaptTo method of the granite SyntheticWorkflowSession